src/App.js

```diff
@@ -1,17 +1,39 @@
 import React, { useState } from "react"
 import './App.css';
 import Button from './component/Button';
-
-
-
-
+import ToDoForm from "./component/TodoForm";
 
 function App() {
-
-}
-
+  const [clickedButton, setClickedButton] = useState(false);
+  const[todos, setTodo] = useState([]);
```

Comment:
Format file. You can install VS code extension Prettier - Code formatted for format code.

`const [todos, setTodo] = useState([]);`